third_party/move/move-vm/runtime/src/storage/module_storage.rs

```diff
@@ -324,6 +345,38 @@ where
             self,
         )?))
     }
+
+    #[cfg(fuzzing)]
+    fn fetch_module(
+        &self,
+        address: &AccountAddress,
+        module_name: &IdentStr,
+    ) -> VMResult<Option<Arc<Module>>> {
+        let id = ModuleId::new(*address, module_name.to_owned());
+
+        // Look up the module in cache, if it is not there, we need to load it
+        let (module, _) = match self.get_module_or_build_with(&id, self)? {
+            Some(module_and_version) => module_and_version,
+            None => return Ok(None),
+        };
+
+        // If module is already verified, return it
+        if module.code().is_verified() {
+            return Ok(Some(module.code().verified().clone()));
+        }
+
+        // Otherwise, create a new Module without verification
+        let runtime_environment = self.runtime_environment();
+        let module = Module::new(
+            runtime_environment.natives(),
+            module.extension().size_in_bytes(),
+            module.code().deserialized().clone(),
+            runtime_environment.struct_name_index_map(),
+        )
+        .map_err(|e| e.finish(Location::Undefined))?;
+
+        Ok(Some(Arc::new(module)))
```

Comment:
Is the intention to also not traverse dependencies?

Reply:
Good catch, I forgot to extend it after testing with samply.